omero/sysadmins/version-requirements.txt

```diff
@@ -0,0 +1,1606 @@
+********************
+Version requirements
+********************
+
+Support levels
+--------------
+
+The following sections use the terminology in the table below to
+describe the support status of a given component, as it progresses
+from being new and not supported, to supported and regularly tested,
+and to finally to being old and no longer supported or tested.
+
+.. tabularcolumns:: |l|l|J|
+
+.. list-table::
+    :header-rows: 1
+
+    * - Level
+      - Meaning
+      - Description
+    * - Upcoming
+      - unsupported/new
+      - New version not yet regularly tested and not officially supported; may or may not work (use at own risk)
+    * - Recommended
+      - supported/optimal
+      - Current version which is regularly tested, confirmed to work correctly, recommended for optimal performance/experience
+    * - Supported
+      - supported/suboptimal
+      - Older version which is tested, confirmed to work correctly, but may not offer optimal performance/experience
```

Comment:
@manics While that's certainly an ideal, it's not really going to be a practical reality.  Given that each Linux distribution has a specific version (or versions) of each component, it's often the case that one or more will be a bit older or newer than our recommendation, e.g. current stable distributions providing PostgreSQL 9.1 while we might wish to mandate 9.2+, same with python, java, ice etc.